index.html

```diff
@@ -0,0 +1,329 @@
+            html
+          </li>
+          <li class="language2">
+            css
+          </li>
+          <li class="language3">
+            javascript
+          </li>
+        </ul>
+        <div class="newmes">
+          <button type="submit" class="go-newmes">See projects</button>
+        </div>
+      </div>
+    </section>
+
+    <section class="works-card-tonic">
+      <img class="snapshort" src="./images/snap3.png" alt="snapshoot" />
+      <div class="left-block-tonic">
+        <div class="first-container-tonic">
+          <h2 class="newme-title-tonic">Tonic</h2>
+          <div class="event2-tonic">
+            <div class="word">
+              <h3>CANOPY</h3>
+            </div>
+            <img src="./images/Counter.png" alt="counter count" />
+            <div class="backend">
+              Back End Dev
+            </div>
+            <img src="./images/Counter.png" alt="counter count" class="dot" />
+            <div class="backend">
+              2015
+            </div>
+          </div>
+        </div>
+        <p class="newme-description-tonic">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="languages-tonic">
+          <li class="language1">
+            html
+          </li>
+          <li class="language2">
+            css
+          </li>
+          <li class="language3">
+            javascript
+          </li>
+        </ul>
+        <div class="newmes-tonic">
+          <button type="submit" class="go-newmes">See projects</button>
+        </div>
+      </div>
+    </section>
+
+
+    <section class="works-card-tonic">
+      <img class="snapshort" src="./images/snap4.png" alt="snapshoot" />
+      <div class="left-block">
+        <div class="first-container">
+          <h2 class="newme-title">
+            Multi-Post Stories<br />
+
+          </h2>
+          <div class="event2-tonic">
+            <div class="word">
+              <h3>CANOPY</h3>
+            </div>
+            <img src="./images/Counter.png" alt="counter count" />
+            <div class="backend">
+              Back End Dev
+            </div>
+            <img src="./images/Counter.png" alt="counter count" class="dot" />
+            <div class="backend">
+              2015
+            </div>
+          </div>
+        </div>
+        <p class="newme-description-tonic">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="languages">
+          <li class="language1">
+            html
+          </li>
+          <li class="language2">
+            css
+          </li>
+          <li class="language3">
+            javascript
+          </li>
+        </ul>
+
+        <div class="newmes">
+          <button type="submit" class="go-newmes">See projects</button>
+        </div>
+        </div>
+        </section>
+        </main>
```

Comment:
- [ ] Great work implementing the changes in your Works structure 💯. However, please kindly ensure to use only 1 `<section>` tag to wrap all the work/project cards and use `<div>` tags to wrap each card as requested by the previous reviewer. 🤓 Currently, you implemented one section for each card. 🔧

Reply:
ok.

I delete:

 </section>


    <section class="works-card-tonic">

![image](https://user-images.githubusercontent.com/35693330/236600585-76b171b2-1adb-4c91-bb5b-38ddd55e46d0.png)